tests/cases/fourslash/completionsJsdocTag.ts

```diff
@@ -5,4 +5,147 @@
 //// * /**/
 //// */
 
+
+// 1x - jsdoc tags are listed when there is more than one whitespace after "*"
+/////**
+//// * /*10*/
+//// */
+////
+/////**
+//// *       /*11*/
+//// */
+////
+
+// 2x - Also, if there are two or more blanks at the beginning of the line
+/////**
+////  /*20*/
+//// */
+////
+/////**
+////      /*21*/
+//// */
+////
+
+// 3x - jsdoc tag names will be listed
+/////** @/*30*/ */
+////
+/////**    @/*31*/ */
+////
+/////**
+//// * @/*32*/
+//// */
+////
+/////**
+//// *       @/*33*/
+//// */
+////
+/////**
+////  @/*34*/
+//// */
+////
+/////**
+////        @/*35*/
+//// */
+////
+/////**
+//// * @pa/*36*/
+//// */
+////
+
+// 4x - jsdoc tag name completions should not occur
+/////**@/*40*/ */
+////
+/////**
+//// *@/*41*/
+//// */
+////
+/////**
+//// * @type {@/*42*/
+//// */
+////
+/////**
+//// +@/*43*/
+//// */
+////
+/////** some description @/*44*/ */
+////
+/////**
+//// * ### jsdoc @/*45*/
+//// */
+////
+/////**
+////@/*46*/
+//// */
+////
+
+// 5x - jsdoc tag completions should not occur
+/////**
+//// */*50*/
+//// */
+////
+/////**
+//// /*51*/
+//// */
+////
+/////**
+/////*52*/
+//// */
+////
+
 verify.completions({ marker: "", includes: { name: "@property", text: "@property", kind: "keyword" } });
+
+
+//
+// test for src/services/completions.ts#getCompletionData.insideComment.hasDocComment (#37546)
+//
+test.markerNames().forEach(marker => {
+    if (marker) {
+        let completionOpt: FourSlashInterface.CompletionsOptions;
+        const n = +marker;
+        switch (n) {
+            // jsdoc tags will be listed when there is more than one whitespace after "*"
+            case 10: case 11:

```

Comment:
very nice way to handle this many cases in one test

Reply:
I was thinking about various cases that should be tested, and it became confusing. 😕
This makes it a little easier by putting together similar cases. 😌